src/hotspot/share/runtime/safepoint.cpp

```diff
@@ -547,33 +538,23 @@ class ParallelSPCleanupTask : public WorkerTask {
     }
   };
 
+  class SafepointCleanupThreadClosure : public ThreadClosure {
```

Comment:
The new name doesn't seem to have much more to do with what it does than did the old name.  The class definition could be moved to the single point of use and just called "Closure" to avoid needing to come up with a good name :)